Feat: deprecate veYFI

rossgalloway opened the pull request:
## Description

- disables the buttons to lock and extend veYFI locks. Still allows locks that are greater than 4 years to be shortened to 4 years.
- Adds prominent CTA about veYFI deprecation.

## Related Issue

YIP-88

## Motivation and Context

YIP-88

## How Has This Been Tested?
locally so far

## Screenshots (if appropriate):

<img width="1282" height="256" alt="image" src="https://github.com/user-attachments/assets/e32e7fda-f63b-4d11-81c1-7e0c707765f4" />

rossgalloway committed:
feat: deprecate veYFI

rossgalloway committed:
fix: revert modify lock changes

rossgalloway committed:
fix: allow lock modification from above max to max

murderteeth reviewed (changes requested):
looks l can still add to my lock, not sure that's intended? and not clear how to reset existing overlock to 4 year vest

0xeye commented on app/components/ViewExtendLockVeYFI.tsx:
```diff
@@ -16,7 +16,8 @@ import type {TNormalizedBN} from '@builtbymom/web3/types';
 export function ExtendLockVeYFI(): ReactElement {
 	const [lockTime, set_lockTime] = useState<TNormalizedBN>(toNormalizedBN(0, 0));
-	const {provider, address, isActive} = useWeb3();
+	// const { provider, address, isActive } = useWeb3();
```
Probably remove comment

0xeye reviewed (approved):
LGTM, maybe remove comments seeing as it's not likely to be re-enabled?

rossgalloway committed:
fix: overlock weeks

rossgalloway committed:
fix: update overlock options and change language

rossgalloway committed:
chore: remove commented code

murderteeth reviewed (changes requested):
is this expected?
- the input box for Lock YFI for veYFI is still enabled, i can click max on it, it populates with loose YFI in my wallet. Approve is disabled, everything else except for Exit is disabled
- my wallet currently has a 512 week lock. i dont see a way to lower that to 4 years

rossgalloway committed:
chore: disable input boxes

rossgalloway committed:
feat: remove overlock, limit interactions

rossgalloway commented:
> is this expected?
> 
> - the input box for Lock YFI for veYFI is still enabled, i can click max on it, it populates with loose YFI in my wallet. Approve is disabled, everything else except for Exit is disabled
> 
> - my wallet currently has a 512 week lock. i dont see a way to lower that to 4 years

- I have removed interactions with the Lock section for everyone and greyed it out.
- if you have a lock >208 weeks, you will now be shown the modify lock section without needing to manually turn on overlocking. You should now be able to reduce a lock to the minimum 209 weeks.
- removed the overLocking toggle.

rossgalloway committed:
fix: lint

murderteeth reviewed (changes requested):
this works. but there's still some weirdness under Modify Lock. if i enter a custom amount then move focus (by clicking a different element for example) the amount changes automatically to 209. so if i enter 300, then click Modify, the value immediately changes to 209, then my wallet opens. which feels shady even though it's not. can we just populate the input with 209 and disable it? then the only step for a user is clicking Modify. might be more clear

rossgalloway committed:
fix: limit modify input
